src/coupled/ESM1.5/cable_surface_types.F90

```diff
@@ -0,0 +1,36 @@
+!******************************************************************************
+! This source code is part of the Community Atmosphere Biosphere Land Exchange
+! (CABLE) model. This work is licensed under the CSIRO Open Source Software
+! License Agreement (variation of the BSD / MIT License).You may not use this
+! this file except in compliance with this License. A copy of the License is
+! available at https://trac.nci.org.au/trac/cable/wiki/license.
+!******************************************************************************
+
+MODULE cable_surface_types_mod 
+
+IMPLICIT NONE
```

Comment:
If we can indent as per https://cable.readthedocs.io/en/latest/developer_guide/documentation_guidelines/science_doc/ that would be great.

Reply:
The link isnt very specific about indentation in this case? Although I suspect you mean this (as fixed). It has been a while since I've submitted anything to JULES but I suspect that they might automatically pull it back to the margin. I dont see any benefit to indenting these layers ? I used to, But after trying to read big nested loops like in canopy, I've been converted.